doc/organoleptic.md

```diff
@@ -0,0 +1,24 @@
+# Organoleptic analysis
+
+version = "0.1.0"
+
+## Overview
+
+this document outlines operations involved in organoleptic testing
+
+## Prerequisites
+
+- Samples
+- Aestetically pleasing containers allowing for flavor development
+- Density measurement tool (optional)
+
+## Procedure
+
+1. Gently pour samples into separate containers, trying to avoid resuspending of yeast
+2. Measure density of samples (only one could be used if convergence of density-dependant parameter was demonstrated otherwise)
+3. Analyze samples smell; record observations
+4. Analyze samples taste; record observations
+5. Discard excesses; recycle yeast mass.
```

Comment:
**not** recycle, use a different word here

Reply:
```suggestion
5. Discard excesses; repurpose remaining yeast mass (for example, for baking).
```